cmd/bridge/main.go

```diff
@@ -211,42 +205,42 @@ func main() {
 		var err error
 
 		if srv.DexClient, err = auth.NewDexClient(*fDexAPIHost, *fDexClientCAFile, *fDexClientCertFile, *fDexClientKeyFile); err != nil {
-			log.Fatalf("Failed to create a Dex API client: %v", err)
+			klog.Fatalf("Failed to create a Dex API client: %v", err)
 		}
 	}
 
 	var secureCookies bool
 	if baseURL.Scheme == "https" {
 		secureCookies = true
-		log.Info("cookies are secure!")
+		klog.Info("cookies are secure!")
```

Comment:
We want to update this to something like:

```go
klog.V(2).Info("cookies are secure!")
```
The rule of thumb in the operator was `.V(4)` or higher for logs that are for developer debugging (most).

Reply:
Yep, missed that one. Thanks!

Reply:
Must have been looking at some outdated code, that is fixed now.

Reply:
Ah, not squashed, yeah looking at prev commit.